siac/main.go

```diff
@@ -11,7 +11,6 @@ import (
 
 	"github.com/bgentry/speakeasy"
 	"github.com/spf13/cobra"
-
```

Comment:
this newline should be here, we like to have a newline between the non-`NebulousLabs` dependencies